smartmin/csv_imports/models.py

```diff
@@ -8,8 +10,16 @@
 from smartmin.models import SmartModel
 
 
+def generate_file_path(instance, filename):
```

Comment:
here I want to make sure the length is not exceeding 64 to prevent https://sentry.io/nyaruka/textit/issues/178987795/

yep I was going to investigate how the uniqueness of name is done on django